app/views/users/index.html.erb

```diff
@@ -7,7 +7,7 @@
     </div>
     <div class="lg-12-3">
       <% unless current_user %>
-        <%= link_to user_omniauth_authorize_path(:github), class: "btn btn-github" do %>
```

Comment:
@msdundar Bahsettiğin bug gerçekten de varmış, sıfırdan kullanıcı açıp deneyince fark ettim sağolasın. Fixini yolladım (https://github.com/rubytr/ruby-tr/pull/62), istersen merge'leyelim onu da bundan sonra